spec/booking_system_spec.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'spec_helper'
+require_relative '../lib/booking_system'
+
+describe "what the hotel booking system does" do
+  before do
+    @room_number = 12
+    @check_in = Date.new(2018, 8, 24)
+    @check_out = @check_in + 5
+    @booking_system = Hotel::BookingSystem.new
+  end
+
+  it "create a class for the booking system" do
+    @booking_system.must_be_kind_of Hotel::BookingSystem
+  end
+
+  it "books a new room" do
+    new_reservation = @booking_system.book_room(@room_number, @check_in, @check_out)
+    new_reservation.must_be_kind_of Hotel::Reservation
+
+    @booking_system.reservations.length.must_equal 1
+  end
```

Comment:
I looks like you have stumbled on one of the secrets of minitest: wrapping values to assert on in `expect` isn't required.

However you should make a habit of doing so, for two reasons:
1. Most other testing frameworks (e.g. Jest for JavaScript, UnitTest for Python) do require something like an `expect`
1. Minitest is going to remove support for leaving `expect` out in the next major release